src/pet/dto/create-pet.input.ts

```diff
@@ -0,0 +1,30 @@
+import { InputType, Field } from '@nestjs/graphql';
+import { IsEnum, IsOptional, IsString, IsUrl } from 'class-validator';
+
+@InputType()
+export class CreatePetInput {
+  @Field(() => String, { description: 'Example field (placeholder)' })
+  @IsString()
+  name: string;
+
+  @Field(() => String, { description: 'Example field (placeholder)' })
+  @IsUrl()
+  photo: string;
+
+  @Field(() => String, { description: 'Example field (placeholder)' })
+  specie: string;
+
+  @Field(() => String, { description: 'Example field (placeholder)' })
+  race: string;
+
+  @Field(() => String, { description: 'Example field (placeholder)' })
+  @IsEnum({ M: 'M', F: 'F' })
+  @IsOptional()
+  genre?: 'M' | 'F';
+
+  @Field(() => String, { description: 'Example field (placeholder)' })
+  age: number;
+
+  @Field(() => String, { description: 'Example field (placeholder)' })
```

Comment:
es mejor crear en la descripcion algo referente a la variable ya que esto nos servira para la documentacion en el momento de hacer las peticiones

Reply:
Regarding your corrections, i did:
- "breed" instead of "race"
- URL instead of "string" in photo
- putting accurate descriptions related to the properties on "create-pet-input".
- setting id to string instead of number.